src/Component/Configuration/Loader/BabelLoader.php

```diff
@@ -28,6 +28,12 @@ public static function applyConfiguration(NodeBuilder $node_builder)
         $node_builder
             ->arrayNode('babel')
                 ->canBeDisabled()
+                ->addDefaultsIfNotSet()
+                ->children()
+                    ->arrayNode('presets')
```

Comment:
addDefaultsIfNotSet() is unnecessary.
I removed it and added some tests for a case nothing is specified.
Please check it.